scripts/link-validator.conf

```diff
@@ -39,7 +35,7 @@ site-link-validator {
     "http://www.thedevpiece.com/"
     # genereated by @apidoc
     "http://pravega.io/"
-    "http://www.scala-lang.org/api/2.13.8/scala/concurrent/Future.html"
-    "http://www.scala-lang.org/api/2.13.8/scala/util/Try.html"
+    "http://www.scala-lang.org/api/2.13.10/scala/concurrent/Future.html"
+    "http://www.scala-lang.org/api/2.13.10/scala/util/Try.html"
```

Comment:
can these be https://www.scala-lang.org/api/2.13.x/scala/concurrent/Future.html ?

Reply:
Sadly not. The generated links contain the exact version.